spec/spec_helper.rb

```diff
@@ -23,7 +21,6 @@
   config.mock_with :rspec do |mocks|
     mocks.verify_partial_doubles = true
   end
```

Comment:
I think you can just delete all of this because there's exactly the same thing after all the DataCleaner things.